WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -73,7 +73,15 @@ decisions = {
 	support_greens = {
 		only_playable = yes
 		potential = {
-			culture_group = latin
+			OR = {
+				culture_group = latin
+				culture = roman
+				has_landed_title = e_wre
+				has_landed_title = e_roman_empire
+				has_title = c_roma
+				has_title = b_rome
+				has_title b_roma
+			}
```

Comment:
If non-Romance characters controlled Rome and the Hippodrome is built when they take over, it's absurd that they would not be able to participate. They govern Rome and thus oversee it, so why wouldn't they take part? This only applies to those that directly control Rome itself.

What's more, non-Romance characters cannot restore the Hippodrome, because of that. This means that it can only occur in very special, very specific circumstances.